### DISCUSSION TITLE
TShock + TModLoader

### DISCUSSION THREAD

**Discussion by Darskiy** [Pryaxis/TShock#2446] [Q&A] [answered]
What is the current situation with this compilation?

**Comment by hakusaro** [comment, marked as answer]
There is no situation or compilation.

**Reply by Darskiy** [reply]
> There is no situation or compilation.

So is there any support planned?

**Reply by drunderscore** [reply]
> > There is no situation or compilation.
> 
> So is there any support planned?

I don't know any contributors that have such support planned -- I certainly don't, I can't think of any good reason someone would want this. ~~Nor do I think the Pryaxis team would be willing to accept such changes.~~ ~~If you really need something like this, make a plugin.~~ Above is my opinion.

**Reply by hakusaro** [reply]
> Nor do I think the Pryaxis team would be willing to accept such changes.

We actually _would_. The appropriate place would be OTAPI. The TModLoader team previously wasn't interested in working with integrating with OTAPI, when death reached out to them to work on it. 🤷🏼‍♀️
